docs/root/configuration/http_filters/grpc_json_transcoder_filter.rst

```diff
@@ -127,6 +127,7 @@ gRPC or RESTful JSON requests to localhost:51051.
                   always_print_primitive_fields: true
                   always_print_enums_as_ints: false
                   preserve_proto_field_names: false
+                ignored_query_parameters: ["key"]
```

Comment:
I don't think you need this for a simple example?

Reply:
Done.